### PR TITLE
line 5,6 and 7 added by aamir6582

### PR DESCRIPTION
**Pull request by aamir6582**

### CODE REVIEW

File: file1 [hunk 1]
```diff
@@ -2,3 +2,4 @@ This is line 1
 This is line 2
 This is line 3
 This  is line 4
+line 5 written by aamir6582
```
Comment (aamirusman1) [inline file1:5]: make changes in line 5

File: file1 [hunk 2]
```diff
@@ -3,3 +3,5 @@ This is line 2
 This is line 3
 This  is line 4
 line 5 written by aamir6582
+line 6 written by aamir6582
```
Comment (aamirusman1) [inline file1:6]: make changes in line 6

File: file1 (hunk 1 above)
Comment (aamir6582) [reply file1]: done

File: file1 (hunk 2 above)
Comment (aamir6582) [reply file1]: changes in line 6 done

**Pull request merged**